src/test/java/org/folio/rest/utils/ReportResourceClient.java

```diff
@@ -1,30 +1,59 @@
 package org.folio.rest.utils;
 
-import static java.lang.String.format;
+import java.util.List;
 
 import org.folio.HttpStatus;
 
 import io.restassured.response.Response;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
 
 public class ReportResourceClient extends ResourceClient {
   public ReportResourceClient(String baseUri) {
     super(baseUri);
   }
 
-  public Response getByDateInterval(String startDate, String endDate) {
-    return getByDateInterval(startDate, endDate, HttpStatus.HTTP_OK);
+  public Response getFeeFineRefundReports(String startDate, String endDate) {
+    return getFeeFineRefundReports(startDate, endDate, null, HttpStatus.HTTP_OK);
   }
 
-  public Response getByDateInterval(String startDate, String endDate, HttpStatus expectedStatus) {
-    return getByParameters(format("startDate=%s&endDate=%s", startDate, endDate),
-      expectedStatus);
+  public Response getFeeFineRefundReports(String startDate, String endDate,
+    List<String> ownerIds) {
+    return getFeeFineRefundReports(startDate, endDate, ownerIds, HttpStatus.HTTP_OK);
   }
 
-  public Response getByParameters(String parameters, HttpStatus expectedStatus) {
-    return okapiClient.get(format("%s?%s", baseUri, parameters))
+  public Response getFeeFineRefundReports(String startDate, String endDate, List<String> ownerIds,
+    HttpStatus expectedStatus) {
+
+    return okapiClient.post(baseUri, createRefundReportRequest(startDate, endDate, ownerIds))
+      .then()
+      .statusCode(expectedStatus.toInt())
+      .extract()
+      .response();
+  }
+
+  public Response getFeeFineRefundReports(String startDate, String endDate,
+    HttpStatus expectedStatus) {
+
+    return okapiClient.post(baseUri, createRefundReportRequest(startDate, endDate, null))
       .then()
       .statusCode(expectedStatus.toInt())
       .extract()
       .response();
   }
```

Comment:
remove it and call getFeeFineRefundReports(String startDate, String endDate, List<String> ownerIds,
    HttpStatus expectedStatus) with null value for owners